index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Luis's Portfolio</title>
+</head>
+<body>
+  <div class="wrapper">
+    <header>
+      <nav class="flex-nav">
+        <ul>
+          <li class="logo"> <a href="#">My Logo</a></li>
+          <li class="menu-item">Portfolio</li>
+          <li class="menu-item">About</li>
+          <li class="menu-item">Contact</li>
+          <li class="hamburger-menu">☰</li>
+        </ul>
+    </nav>
+  </header>
+
+    <section class="content">
+      <h1 class="title">
+        I'm Luis <br>Glad to see you!
+      </h1>
+      <p class="descr">I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <h2 class="connect"> <a href="#">Let's connect</a></h2>
+      <ul class="socials">
+        <li><a href="https://twitter.com/lujimhe" target="_blank" rel="noopener noreferrer">
+          <img src="https://tinyurl.com/2kabbz4x" alt="Twitter logo" ></a></li>
+        <li><a href="https://www.linkedin.com/in/lujim/" target="_blank" rel="noopener noreferrer">
+          <img src="https://tinyurl.com/yrsm2bzk" alt="Linkedin logo"></a></li>
+        <li><a href="https://medium.com/@lu.jim" target="_blank" rel="noopener noreferrer">
+          <img src="https://tinyurl.com/rrt889rt" alt="Medium logo"></a></li>
+        <li><a href="https://github.com/lu-jim" target="_blank" rel="noopener noreferrer">
+          <img src="https://tinyurl.com/githublogo" alt="Github logo"></a></li>
+        <li><a href="https://angel.co" target="_blank" rel="noopener noreferrer">
+          <img src="https://tinyurl.com/jnb8s2ds" alt="AngelList logo"></a></li>
+      </ul>
```

Comment:
- [ ] The social icons are still missing, I would recommend downloading the icons and save them in an `/assets` folder and then change the source of the images.
    - For example:
    ```
    # /assets
          twitter-icon.png
    # HTML
          <img src="./assets/twitter-icon.png" />
    ```
    - Remember that the file names can't have spaces. 🤓 
    - Your contact section currently see like this: 
    ![imagen](https://user-images.githubusercontent.com/60789317/133856501-b16453d2-b275-49a0-8c9d-d5ef861c64cc.png)